client-course-schedulizer/src/utilities/services/addSectionService.ts

```diff
@@ -40,13 +34,13 @@ export interface SectionInput {
   globalMax?: Section["globalMax"];
   halfSemester: Half;
   instructionalMethod: Section["instructionalMethod"];
-  instructor: Instructor;
+  instructor: Instructor[];
   intensiveSemester: Intensive;
   localMax?: Section["localMax"];
   location: string;
   name: Course["name"];
   number: Course["number"];
-  prefix: Prefix;
+  prefix: Prefix[];
```

Comment:
What are the implications of making this change to arrays? Will this affect how we parse things when importing and exporting schedules?

Reply:
There shouldn't be any implications since prefixes is already an array in the `Course` interface. This change is just for the form input interface and makes it so you know longer have to split on commas.